src/schema/v2/partner/partners.ts

```diff
@@ -245,7 +245,10 @@ export const PartnersConnection: GraphQLFieldConfig<void, ResolverContext> = {
     }
 
     // Removes null/undefined values from options
-    const cleanedOptions = pickBy(clone(options), identity)
+    const cleanedOptions = pickBy(
+      clone(options),
+      (option) => option !== null && option !== undefined
+    )
```

Comment:
Replacing `identity` with `(option) => option !== null && option !== undefined` to not filter out options set to `false`.